snforge_std/src/lib.cairo

```diff
@@ -33,13 +36,4 @@ use cheatcodes::stop_warp;
 use cheatcodes::start_mock_call;
 use cheatcodes::stop_mock_call;
 
-
 mod io;
-
-use io::file_operations::File;
-use io::file_operations::FileTrait;
-use io::file_operations::read_txt;
-use io::file_operations::read_json;
-use io::file_operations::FileParser;
-
-use io::forge_print::PrintTrait;
```

Comment:
Additionally to these changes, lines like this should be added to `io.cairo` file:

```cairo
use io::file_operations::File;
use io::file_operations::FileTrait;
// ... etc
```

This way you will be able to use imports like this in tests code:

```cairo
use io::File;
```

So we are enforcing using the module name in path, but do not force the user to write the whole file name for each function.

Reply:
You can check out my code in this PR: https://github.com/foundry-rs/starknet-foundry/pull/705/files#diff-65daba458203fdcab6033fc02561c249c5c8f796ba4938e1741185434837e9fe to see how this can be approached.

Reply:
Here's an example usage: https://github.com/foundry-rs/starknet-foundry/pull/705/files#diff-ed0161129846964bde0ae2888f9ec23fbed5498cb4e733d8f6cfb9118107b678R11